### ISSUE TITLE
panic when resizing window quickly

### ISSUE THREAD

**Issue by F-bh** [iced-rs/iced#943]
When very quickly resizing the window while the options of a pick list  are still opened wgpu panics.
Built with release version 0.3.0.
I had no minimal window size set in my application so that may be a possible solution to the problem.
I think it's not a big problem, since most people probaly don't resize their windows that quickly, but I hope reporting it helps anyways 
Tried on Arch with x11, and Windows 10 (seems much harder to reproduce on Windows); both also as release builds
here is the error message and a full stacktrace:
```
Caused by:
    In a RenderPass
      note: encoder = `iced_wgpu encoder`
    In a set_scissor_rect command
    Invalid ScissorRect parameters


thread 'main' panicked at 'Handling wgpu errors as fatal by default', /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.1/src/backend/direct.rs:1896:5
stack backtrace:
   0:     0x559b0b62c010 - std::backtrace_rs::backtrace::libunwind::trace::ha5edb8ba5c6b7a6c
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/../../backtrace/src/backtrace/libunwind.rs:90:5
   1:     0x559b0b62c010 - std::backtrace_rs::backtrace::trace_unsynchronized::h0de86d320a827db2
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x559b0b62c010 - std::sys_common::backtrace::_print_fmt::h97b9ad6f0a1380ff
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:67:5
   3:     0x559b0b62c010 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h14be7eb08f97fe80
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:46:22
   4:     0x559b0b64fd6f - core::fmt::write::h2ca8877d3e0e52de
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/fmt/mod.rs:1094:17
   5:     0x559b0b627c35 - std::io::Write::write_fmt::h64f5987220b618f4
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/io/mod.rs:1584:15
   6:     0x559b0b62e12b - std::sys_common::backtrace::_print::h7f1a4097308f2e0a
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:49:5
   7:     0x559b0b62e12b - std::sys_common::backtrace::print::h1f799fc2ca7f5035
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:36:9
   8:     0x559b0b62e12b - std::panicking::default_hook::{{closure}}::hf38436e8a3ce1071
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:208:50
   9:     0x559b0b62dbfd - std::panicking::default_hook::he2f8f3fae11ed1dd
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:225:9
  10:     0x559b0b62e73d - std::panicking::rust_panic_with_hook::h79a18548bd90c7d4
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:591:17
  11:     0x559b0ac043e3 - std::panicking::begin_panic::{{closure}}::h4a8de5be05e0e211
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/panicking.rs:520:9
  12:     0x559b0ac075e9 - std::sys_common::backtrace::__rust_end_short_backtrace::h7b45fe68b232f731
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys_common/backtrace.rs:141:18
  13:     0x559b0ac04319 - std::panicking::begin_panic::h554138e10e47ef90
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/panicking.rs:519:12
  14:     0x559b0a61f73c - wgpu::backend::direct::default_error_handler::ha4bf694ff4fd2bbb
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.1/src/backend/direct.rs:1896:5
  15:     0x559b0a51dbdc - core::ops::function::Fn::call::heb9ed47f648b2295
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:70:5
  16:     0x559b0a551722 - <alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call::h56f5417e65bfb5f6
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1560:9
  17:     0x559b0a61f631 - wgpu::backend::direct::ErrorSinkRaw::handle_error::hf3bc427adb5247ad
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.1/src/backend/direct.rs:1883:9
  18:     0x559b0a612610 - wgpu::backend::direct::Context::handle_error::h5309c10bbc40882c
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.1/src/backend/direct.rs:109:9
  19:     0x559b0a61f1d5 - <wgpu::backend::direct::Context as wgpu::Context>::command_encoder_end_render_pass::h6de10a3bf2e7b9f2
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.1/src/backend/direct.rs:1739:13
  20:     0x559b0a7882e7 - <wgpu::RenderPass as core::ops::drop::Drop>::drop::hc3f315744c87ba9f
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.1/src/lib.rs:2486:13
  21:     0x559b0a3f27f7 - core::ptr::drop_in_place<wgpu::RenderPass>::h2af6292f5b2a0942
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:192:1
  22:     0x559b0a3e905c - iced_wgpu::quad::Pipeline::draw::h0830a72e77106f20
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced_wgpu-0.4.0/src/quad.rs:275:13
  23:     0x559b0a393864 - iced_wgpu::backend::Backend::flush::h576ae49126f7448f
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced_wgpu-0.4.0/src/backend.rs:116:13
  24:     0x559b0a21079f - iced_wgpu::backend::Backend::draw::ha6c7fbe509ee01e2
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced_wgpu-0.4.0/src/backend.rs:82:13
  25:     0x559b0a15e164 - <iced_wgpu::window::compositor::Compositor as iced_graphics::window::compositor::Compositor>::draw::he400b60ee8da6ae3
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced_wgpu-0.4.0/src/window/compositor.rs:156:33
  26:     0x559b0a1e822b - iced_winit::application::run_instance::{{closure}}::h50d5ea1a1fcbf693
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced_winit-0.3.0/src/application.rs:350:45
  27:     0x559b0a1d5d8b - <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::hf04063971c70464c
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/future/mod.rs:80:19
  28:     0x559b0a1ea662 - iced_winit::application::run::{{closure}}::hd8a42d2740b69e01
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced_winit-0.3.0/src/application.rs:193:24
  29:     0x559b0a20777e - winit::platform_impl::platform::sticky_exit_callback::h7c69a71bb0531b08
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.24.0/src/platform_impl/linux/mod.rs:736:5
  30:     0x559b0a13fe66 - winit::platform_impl::platform::x11::EventLoop<T>::run_return::hdcffdd72ff9130b9
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.24.0/src/platform_impl/linux/x11/mod.rs:320:21
  31:     0x559b0a1427a4 - winit::platform_impl::platform::x11::EventLoop<T>::run::hb69f9987cb93456a
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.24.0/src/platform_impl/linux/x11/mod.rs:398:9
  32:     0x559b0a207400 - winit::platform_impl::platform::EventLoop<T>::run::h004a0df00057fe2d
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.24.0/src/platform_impl/linux/mod.rs:652:56
  33:     0x559b0a189ea6 - winit::event_loop::EventLoop<T>::run::h7d836c8f2a7fa615
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.24.0/src/event_loop.rs:154:9
  34:     0x559b0a1e9de0 - iced_winit::application::run::h97881f64618a8ef6
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced_winit-0.3.0/src/application.rs:168:5
  35:     0x559b0a13f04b - iced::application::Application::run::hff089542f2762e0f
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/iced-0.3.0/src/application.rs:219:16
  36:     0x559b0a15b567 - Oxidized_Soundboard::main::hb31c60935c89580e
                               at /home/fbh/Coding/Rust Projekte/Oxidized-Soundboard/src/main.rs:19:5
  37:     0x559b0a245452 - core::ops::function::FnOnce::call_once::hb3fd58c075615131
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
  38:     0x559b0a164c9a - std::sys_common::backtrace::__rust_begin_short_backtrace::h91d3a29fe6e55a10
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys_common/backtrace.rs:125:18
  39:     0x559b0a135e96 - std::rt::lang_start::{{closure}}::ha7216d5828a8fb5c
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/rt.rs:49:18
  40:     0x559b0b62eb3a - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::hc4354216bf39217c
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/ops/function.rs:259:13
  41:     0x559b0b62eb3a - std::panicking::try::do_call::hb68eb312780385cf
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:379:40
  42:     0x559b0b62eb3a - std::panicking::try::h22b8e08595060b8b
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:343:19
  43:     0x559b0b62eb3a - std::panic::catch_unwind::hc64f1a6a0e71b1fc
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panic.rs:431:14
  44:     0x559b0b62eb3a - std::rt::lang_start_internal::h4461fc58637f04f8
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/rt.rs:34:21
  45:     0x559b0a135e70 - std::rt::lang_start::h91d70509eb7ce15a
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/rt.rs:48:5
  46:     0x559b0a15d29c - main
  47:     0x7f00825b0b25 - __libc_start_main
  48:     0x559b0a0b21de - _start
  49:                0x0 - <unknown>

Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)
```

**Comment by hecrj** [comment]
Could you test while using `master`? I believe #818 should have fixed this.

**Comment by F-bh** [comment]
I might have found another issue trying, cargo build seems to fail for  web assembly with this web-sys conflict:
```
error: failed to select a version for `web-sys`.
    ... required by package `wgpu v0.9.0`
    ... which is depended on by `iced_wgpu v0.4.0 (https://github.com/hecrj/iced.git#4994d34a)`
    ... which is depended on by `iced v0.3.0 (https://github.com/hecrj/iced.git#4994d34a)`
    ... which is depended on by `KEK v0.1.0 (/home/fbh/CLionProjects/KEK)`
versions that meet the requirements `=0.3.50` are: 0.3.50

all possible versions conflict with previously selected packages.

  previously selected package `web-sys v0.3.46`
    ... which is depended on by `iced_web v0.4.0 (https://github.com/hecrj/iced.git#4994d34a)`
    ... which is depended on by `iced v0.3.0 (https://github.com/hecrj/iced.git#4994d34a)`
    ... which is depended on by `KEK v0.1.0 (/home/fbh/CLionProjects/KEK)`

failed to select a version for `web-sys` which could resolve this conflict
```

**Comment by F-bh** [comment]
And if I build master on linux the resizing issue still persists but the stack trace seems to have changed:
```
thread 'main' panicked at 'Handling wgpu errors as fatal by default', /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.9.0/src/backend/direct.rs:1978:5
stack backtrace:
   0:     0x55b9dade26d0 - std::backtrace_rs::backtrace::libunwind::trace::ha5edb8ba5c6b7a6c
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/../../backtrace/src/backtrace/libunwind.rs:90:5
   1:     0x55b9dade26d0 - std::backtrace_rs::backtrace::trace_unsynchronized::h0de86d320a827db2
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x55b9dade26d0 - std::sys_common::backtrace::_print_fmt::h97b9ad6f0a1380ff
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:67:5
   3:     0x55b9dade26d0 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h14be7eb08f97fe80
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:46:22
   4:     0x55b9dae05d0f - core::fmt::write::h2ca8877d3e0e52de
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/fmt/mod.rs:1094:17
   5:     0x55b9dadde465 - std::io::Write::write_fmt::h64f5987220b618f4
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/io/mod.rs:1584:15
   6:     0x55b9dade47eb - std::sys_common::backtrace::_print::h7f1a4097308f2e0a
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:49:5
   7:     0x55b9dade47eb - std::sys_common::backtrace::print::h1f799fc2ca7f5035
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/sys_common/backtrace.rs:36:9
   8:     0x55b9dade47eb - std::panicking::default_hook::{{closure}}::hf38436e8a3ce1071
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:208:50
   9:     0x55b9dade42bd - std::panicking::default_hook::he2f8f3fae11ed1dd
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:225:9
  10:     0x55b9dade4dfd - std::panicking::rust_panic_with_hook::h79a18548bd90c7d4
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:591:17
  11:     0x55b9dad3dfb3 - std::panicking::begin_panic::{{closure}}::h80f6d7abca012284
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/panicking.rs:520:9
  12:     0x55b9dad3fd59 - std::sys_common::backtrace::__rust_end_short_backtrace::h6c514375a909adab
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys_common/backtrace.rs:141:18
  13:     0x55b9dad3dee9 - std::panicking::begin_panic::h4951ed7d9d7e3972
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/panicking.rs:519:12
  14:     0x55b9d9de9962 - wgpu::backend::direct::default_error_handler::h86f9224d36a56a82
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.9.0/src/backend/direct.rs:1978:5
  15:     0x55b9da0743fc - core::ops::function::Fn::call::h8e0c2990d3b33702
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:70:5
  16:     0x55b9da09b8a2 - <alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call::h5050d81442baa8b0
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1560:9
  17:     0x55b9d9de9781 - wgpu::backend::direct::ErrorSinkRaw::handle_error::hc241b831a5be2c3c
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.9.0/src/backend/direct.rs:1965:9
  18:     0x55b9d9ddf3b1 - wgpu::backend::direct::Context::handle_error::h9ed09e7494f886d3
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.9.0/src/backend/direct.rs:93:9
  19:     0x55b9d9de9005 - <wgpu::backend::direct::Context as wgpu::Context>::command_encoder_end_render_pass::h927c0229a37f2e4d
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.9.0/src/backend/direct.rs:1764:13
  20:     0x55b9d9fc025a - <wgpu::RenderPass as core::ops::drop::Drop>::drop::h6047204ecc80b9e5
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.9.0/src/lib.rs:2629:13
  21:     0x55b9d9c10047 - core::ptr::drop_in_place<wgpu::RenderPass>::hd1382e14a75619fc
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:192:1
  22:     0x55b9d9c0e62e - wgpu_glyph::pipeline::draw::h35473589035ba1b8
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu_glyph-0.13.0/src/pipeline.rs:404:1
  23:     0x55b9d9c0cb58 - wgpu_glyph::pipeline::Pipeline<()>::draw::hdbe62ebfb8b0a930
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu_glyph-0.13.0/src/pipeline.rs:53:9
  24:     0x55b9d9b8a152 - wgpu_glyph::GlyphBrush<(),F,H>::draw_queued_with_transform_and_scissoring::he4f03e6ac20aeefb
                               at /home/fbh/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu_glyph-0.13.0/src/lib.rs:316:9
  25:     0x55b9d9b7a74a - iced_wgpu::text::Pipeline::draw_queued::h66fadf34a8afa7b6
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/wgpu/src/text.rs:75:9
  26:     0x55b9d9b90542 - iced_wgpu::backend::Backend::flush::hd51063cdfaf5b6f5
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/wgpu/src/backend.rs:230:13
  27:     0x55b9d9acaaff - iced_wgpu::backend::Backend::draw::hf6b1c8a0ea3f23e5
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/wgpu/src/backend.rs:82:13
  28:     0x55b9d9ae8fee - <iced_wgpu::window::compositor::Compositor as iced_graphics::window::compositor::Compositor>::draw::haa973716d0f30c86
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/wgpu/src/window/compositor.rs:169:33
  29:     0x55b9d9aa0b23 - iced_winit::application::run_instance::{{closure}}::h4c8648af0ac16a17
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/winit/src/application.rs:360:45
  30:     0x55b9d9b019e9 - <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h21898cc5fbd37a82
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/future/mod.rs:80:19
  31:     0x55b9d9aa30b2 - iced_winit::application::run::{{closure}}::h2764d0e50f43ead2
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/winit/src/application.rs:193:24
  32:     0x55b9d9aaec5e - winit::platform_impl::platform::sticky_exit_callback::hdade65b443adb016
                               at /home/fbh/.cargo/git/checkouts/winit-57d3141eaf559308/44a9a6f/src/platform_impl/linux/mod.rs:754:5
  33:     0x55b9d9b0dd00 - winit::platform_impl::platform::x11::EventLoop<T>::run_return::h84862fc8757b28c8
                               at /home/fbh/.cargo/git/checkouts/winit-57d3141eaf559308/44a9a6f/src/platform_impl/linux/x11/mod.rs:307:21
  34:     0x55b9d9b107d4 - winit::platform_impl::platform::x11::EventLoop<T>::run::h38a5cc2d38b8c5ec
                               at /home/fbh/.cargo/git/checkouts/winit-57d3141eaf559308/44a9a6f/src/platform_impl/linux/x11/mod.rs:385:9
  35:     0x55b9d9aae8e0 - winit::platform_impl::platform::EventLoop<T>::run::h8a3fd4d176411802
                               at /home/fbh/.cargo/git/checkouts/winit-57d3141eaf559308/44a9a6f/src/platform_impl/linux/mod.rs:670:56
  36:     0x55b9d9aade66 - winit::event_loop::EventLoop<T>::run::h118e063578a6743d
                               at /home/fbh/.cargo/git/checkouts/winit-57d3141eaf559308/44a9a6f/src/event_loop.rs:154:9
  37:     0x55b9d9aa2888 - iced_winit::application::run::he8dbd355f6691cc8
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/winit/src/application.rs:168:5
  38:     0x55b9d9b0c16b - iced::application::Application::run::h4b2c04a2b835b6c2
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/src/application.rs:219:16
  39:     0x55b9d9b0c51c - iced::sandbox::Sandbox::run::h8c9ec9ba5b0dd1f7
                               at /home/fbh/.cargo/git/checkouts/iced-9e73d2fbe1fce35a/4994d34/src/sandbox.rs:145:9
  40:     0x55b9d9b1359b - KEK::main::h0437f8f40ddf79e9
                               at /home/fbh/CLionProjects/KEK/src/main.rs:4:5
  41:     0x55b9d9b2100b - core::ops::function::FnOnce::call_once::h2bc2fca0848bd040
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
  42:     0x55b9d9b560ee - std::sys_common::backtrace::__rust_begin_short_backtrace::hbd5b14ee217c1d5c
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys_common/backtrace.rs:125:18
  43:     0x55b9d9af42c1 - std::rt::lang_start::{{closure}}::hbc965e6081741080
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/rt.rs:49:18
  44:     0x55b9dade51fa - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::hc4354216bf39217c
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/ops/function.rs:259:13
  45:     0x55b9dade51fa - std::panicking::try::do_call::hb68eb312780385cf
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:379:40
  46:     0x55b9dade51fa - std::panicking::try::h22b8e08595060b8b
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:343:19
  47:     0x55b9dade51fa - std::panic::catch_unwind::hc64f1a6a0e71b1fc
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panic.rs:431:14
  48:     0x55b9dade51fa - std::rt::lang_start_internal::h4461fc58637f04f8
                               at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/rt.rs:34:21
  49:     0x55b9d9af42a0 - std::rt::lang_start::hcba42ad1adbdb232
                               at /home/fbh/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/rt.rs:48:5
  50:     0x55b9d9b138dc - main
  51:     0x7f4373bd1b25 - __libc_start_main
  52:     0x55b9d9a88cae - _start
  53:                0x0 - <unknown>

Process finished with exit code 101
```

**Comment by rurigk** [comment]
I tried to replicate in Manjaro without success
With multiple displays, change the scale factor, drag the window really fast, resizing the window as fast as i can
#818 fixed the crash for me

I fixed the web-sys problem by deleting the Cargo.lock file and runing cargo build again (keep in mind that i started a new project and maybe you cant do that if you have a project already working)

Also added iced with 
[dependencies]
iced = { git = "https://github.com/hecrj/iced" }

My setup:
OS: Manjaro Linux x86_64 
Kernel: 5.12.16-1-MANJARO 
Resolution: 1920x1080, 1920x1080 
DE: Plasma 5.22.3 
WM: KWin
CPU: AMD Ryzen 7 2700X (16) @ 3.700GHz 
GPU: AMD ATI Radeon RX 570
Memory: 8767MiB / 32099MiB

**Comment by F-bh** [comment]
deleting the cargo.lock file also fixed the web-sys problem for me, but not the crash even though we are on very similar systems:
OS: Manjaro Linux x86_64
Kernel: 5.13.1-3-MANJARO
Resolution: 1920x1080, 1920x1080
DE: Plasma 5.22.3
WM: KWin
CPU: AMD Ryzen 5 3600 (12) @ 3.600GHz
GPU: Nvidia GTX 970
Memory: 4419MiB / 15950MiB

It isn't some Nvidia problem either, since my laptop uses an AMD 3500U APU

**Comment by hecrj** [comment]
This should be fixed by #818.
